mitodl/mitxonline
# Fetch by department when selected

### What are the relevant tickets?
Closes https://github.com/mitodl/hq/issues/3748

### Description (What does it do?)
This PR adds in a `queryIDListString` to the `getNextCoursePage` function which takes in a string of course IDs and appends that as a parameter for` id=` to the end of the API url.  This is set when a department other than `ALL_DEPARTMENTS` is selected and `getNextCoursePage` is called.

This should make it so that on a page with less than the expected numebr of courses already loaded, the missing courses are loaded.


### How can this be tested?
Load up with more than 12 courses. You should have a few in a couple of departments. This will allow you to see if, when you swap to one, it loads after swapping.
